app/pages/lab-pages-editor/components/TasksPage/TasksPage.jsx

```diff
@@ -230,3 +231,21 @@ export default function TasksPage() {
     </div>
   );
 }
+
+function getPreviewEnv() {
```

Comment:
This function feels too important to bury in a file... I'd personally want something like this (with clear exceptions / responsive to our development URL schema) surfaced a bit higher. I'ts own file, part of a helper, or something that makes its purpose more surfaceable.